setup/helpers/abs_helper.rb

```diff
@@ -202,14 +204,32 @@ def get_abs_resource_hosts(hosts_to_request)
   #
   def get_abs_resource_host(host_to_request)
     uri = URI("#{@abs_base_url}/awsdirect")
+    role = host_to_request[:role]
+
     puts "Host_to_request: #{host_to_request}"
+    puts
 
     request_body = get_awsdirect_request_body(host_to_request)
-    response = perform_awsdirect_request(uri, request_body)
+    result = nil
+
+    (1..ABS_MAX_REQUEST_ATTEMPTS).each do |ct|
+      response = perform_awsdirect_request(uri, request_body)
```

Comment:
Should there be a delay between retries?

Reply:
Good call; I will add this setting.

Reply:
Updated.